ansible/roles/kong-api/defaults/main.yml

```diff
@@ -5692,3 +5693,1101 @@ kong_apis:
       config.limit_by: credential
     - name: request-size-limiting
       config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+  - name: tags
+    uris: "{{ discussions_prefix }}/tags"
+    upstream_url: "{{ discussions_mw_service_url }}/tags"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'contentCreate'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ medium_request_size_limit }}"      
+
+  - name: categories
+    uris: "{{ discussions_prefix }}/categories"
+    upstream_url: "{{ discussions_mw_service_url }}/categories"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'contentCreate'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+  - name: notifications
+    uris: "{{ discussions_prefix }}/notifications"
+    upstream_url: "{{ discussions_mw_service_url }}/notifications"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'contentCreate'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+  - name: userDetails
+    uris: "{{ discussions_prefix }}/user/:userslug"
+    upstream_url: "{{ discussions_mw_service_url }}/user/:userslug"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'contentCreate'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+  - name: userUpvoteList
+    uris: "{{ discussions_prefix }}/user/:userslug/upvoted"
```

Comment:
is this URL correct?

Reply:
is :usersulg a variable or a static value?

Reply:
variable

Reply:
As of now it is not supported by our automatoin @vinukumar-vs ! can we move the variable part to the end?